README.md

```diff
@@ -1 +1,35 @@
-# Random Activitygen 
+# RandomActivityGen
+
+This tool provides an easy way to generate semi-realistic traffic demand for [SUMO](http://sumo.sourceforge.net/) by generating a population statistics file (`.stats.xml`) for [ACTIVITYGEN](https://sumo.dlr.de/docs/ACTIVITYGEN.html).
+
+RandomActivityGen works best for city networks, and works very well with a random [NETGENERATE](https://sumo.dlr.de/docs/NETGENERATE.html) network.  
+
+## Usage
+
+**Required input:**
+
+* A network file (`.net.xml`) as generated via [NETCONVERT](https://sumo.dlr.de/docs/NETCONVERT.html) or [NETGENERATE](https://sumo.dlr.de/docs/NETGENERATE.html),
+* A population definition in form of a [statistics file](https://sumo.dlr.de/docs/Demand/Activity-based_Demand_Generation.html) (`stats.xml`), but the only required fields and attributes are:
+```
+<city>
+    <general inhabitants="3500" households="2000"/>
+</city>
+```
+
+More fields can be specified if desired. RandomActivityGen will generate any missing values and populate the statistics file with streets, city gates, and schools.
```

Comment:
Perhaps a small comment that you can overwrite some of the default values for creating schools/city gates (e.g cityGates count), and that you can see these with --help. Else looks good